docs/source/algorithms/SaveINS-v1.rst

```diff
@@ -0,0 +1,46 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+Function to save a .ins file - one of the input files required for the SHELX crystallographic code.
+The file contains information about the sample (spacegroup, lattice parameters and atoms present) and the format
+of the reflections file.
+
+The ``InputWorkspace`` must have a UB set (from which the lattice parameters are retrieved) and a sample material
+(from which the atoms and number of formula units are taken). The spacegroup can be taken given as an input argument to
+the function, or from the crystal structure if one has been set on the workspace.
+
+By default it is assumed that atoms have a ratio of isotopes seen in nature, however if
+``UseNaturalIsotopicAbundances=False`` the scattering scattering cross-sections defined for the isotopes specified in
+the sample material will be used. In this case there are other atomic properties that are output, one of these is the
+covalent radius (which SHELX uses for the purposes of outputting bonds/geometry information) which is set to be
+1 Angstrom (if ``UseNaturalIsotopicAbundances=True`` SHELX will look up the covalent radii of atoms itself).
```

Comment:
Is there a test for this case where `covalent radius` is different depending on `UseNaturalIsotopicAbundances`?

Reply:
So the covalent radius is only ever printed to file when `UseNaturalIsotopicAbundances=False` (one of the numbers in the `SFAC` line) - so this is covered by `test_save_ins_natural_isotopic_abundance_true`

Reply:
That makes sense thanks